vector/v.lidar.correction/main.c

```diff
@@ -34,7 +34,7 @@ int main(int argc, char *argv[])
     int subregion = 0, nsubregions = 0;
     const char *dvr, *db, *mapset;
     char table_name[GNAME_MAX];
-    char xname[GNAME_MAX], xmapset[GMAPSET_MAX];
+    char xname[GNAME_MAX - 5], xmapset[GMAPSET_MAX];
```

Comment:
Why reduce the length of xname? Generally, more space is better?

Why reduce the length of xname even more in ` vector/v.lidar.edgedetection/main.c` and ` vector/v.lidar.growing/main.c` ?

`xname` is always used with `G_name_is_fully_qualified()`

Reply:
See line eg https://github.com/OSGeo/grass/blob/777c4e672be4b6fb4504083736259fdeb7bc162f/vector/v.lidar.correction/main.c#L189
table_name is equal in size to xname

Reply:
Probably better to use `snprintf()` for these too.

Reply:
The problem behind the fixes with reduced xname size have all in common addition of extension to the mapname (which can be `GNAME_MAX`) to the db table name (presently also with size of GNAME_MAX). If there is no limitation to the size of database table name (I couldn't find any), then increasing that seems to be the easier solution.

Reply:
In this case I would prefer to increase the size of database table names. These database table names often consist of the map name plus some more information and are hopefully not limited in size (to `GNAME_MAX`).